pkg/configuration/config.go

```diff
@@ -239,10 +248,9 @@ func AllConfigs() map[string]models.FileScopedOptions {
 
 // Set properties on a scoped config
 func Set(scope string, options map[string]string) {
-	scope = NormalizeScope(scope)
 	var err error
-	if scope, err = utils.ParsePath(scope); err != nil {
-		utils.HandleError(err)
+	if scope, err = NormalizeScope(scope); err != nil {
+		utils.HandleError(err, "Invalid scope")
```

Comment:
Can we provide more human errors? Like `The provided scope does not exist on the file system`.

Reply:
This function doesn't check whether the path exists, it just checks if the path contains all valid characters.